01-intro.Rmd

```diff
@@ -28,6 +28,10 @@ Confirmatory Analysis: to confirm our understanding and analysis of the data | !
 * It is flexible. The world is densely connected, so as long as there is an interesting relationship in your data somewhere, you will find value in graph visualization.
 * It is insightful. Exploring graph data interactively allows users to gain more in-depth knowledge, understand the context and ask more questions, compared to static visualization or raw data.
 
+## Importance of Data Exploration
+"Data exploration tasks are those of examining data without having an a priori understanding of what patterns, information, or knowledge it might contain." [@Data_Exploration_1] While the majority of research has been on the output of and creation of visualizations, but the important aspect of these tasks is to first understand the data that is being presented. As mentioned in [@Data_Exploration_1], the main aspect of exploration is the understanding of perception in comparing two measures in a data source. The contrast between object A and object B shows an ability to compare by saying A is greater than B or the opposite occurs. As a brief introduction, data exploration is the process of finding the best way to pull out an outcome that a specific audience can perceive. By understanding the different types of perceptions and studying data based off of these concepts, data exploration is pivotal in the final visualization process due to taking the audience of the viz into account first and foremost. 
```

Comment:
There are few grammatical mistakes- 
1.) usage of an and a together in the first sentence
2.)In the second line, there is no need for but after the comma
3.) in place of 'or the opposite occurs' , usage of 'vice versa' would be more appropriate.
4.) In the second last line, usage of 'based on' in place of 'based off' would be more appropriate.